.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,195 @@
+
+      - name: Download Atom Data
+        if: steps.atom-data-restore.outputs.cache-hit != 'true'
+        run: |
+          REPO_URL="https://dev.azure.com/tardis-sn/TARDIS/_apis/git/repositories/tardis-refdata"
+          wget -q "$REPO_URL/items?path=atom_data/kurucz_cd23_chianti_H_He.h5&versionType=branch&version=master&resolveLfs=true" -O benchmarks/kurucz_cd23_chianti_H_He.h5
+
+      - name: Save Atom Data
+        uses: actions/cache/save@v3
+        if: ${{ steps.atom-data-restore.cache-hit != 'true'  && always() || false }}
+        id: atom-data-cache-save
+        with:
+          path: benchmarks/kurucz_cd23_chianti_H_He.h5
+          key: atom-data
+
+      - name: Install asv
+        run: pip install asv==0.5.*
+
+      - name: Accpet all asv questions
+        run: asv machine --yes
+
+      - name: Download TARDIS dependencies
+        run: |
+          LOCK_URL="https://github.com/tardis-sn/tardis/raw/${{ env.TARDIS_VER }}/tardis_env3.yml"
+          wget -q "$LOCK_URL"
+
+      - name: Add TARDIS install command to asv config file
+        run: |
+          TARDIS_INSTALL_CMD="pip install git+https://github.com/tardis-sn/tardis.git@${{ env.TARDIS_VER }}"
+          cat <<< $(jq '.build_command |= [$TARDIS_INSTALL_CMD] + .' --arg TARDIS_INSTALL_CMD "${TARDIS_INSTALL_CMD}" asv.conf.json) > asv.conf.json
+
+      - name: Run benchmarks for last 5 commits if not PR
+        if: github.event_name != 'pull_request'
+        run: |
+          git log -n 5 --pretty=format:"%H" >> tag_commits.txt
+          asv run -v HASHFILE:tag_commits.txt | tee asv-output.log  
+          if grep -q failed asv-output.log; then 
+            echo "Some benchmarks have failed!"
+            exit 1
+          fi
+
+      - name: Generate Graphs and HTML
+        if: github.event_name != 'pull_request'
+        run: asv publish
+
+      - name: Delete env files
+        run: rm -r .asv/env
+
+      - name: Create results html on dest repo
+        if: github.event_name != 'pull_request'
+        continue-on-error: true
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.BOT_TOKEN }}
+        with:
+          source-directory: ".asv/html"
+          destination-github-username: "tardis-sn"
+          destination-repository-name: "stardis-benchmarks"
+          user.email: tardis.sn.bot@gmail.com
+          target-branch: main
+
+      - name: Push misc data to dest repo
+        if: github.event_name != 'pull_request'
+        continue-on-error: true
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.BOT_TOKEN }}
+        with:
+          source-directory: ".asv/results"
+          destination-github-username: "tardis-sn"
+          destination-repository-name: "stardis-benchmarks"
+          user.email: tardis.sn.bot@gmail.com
+          target-branch: main
+          target-directory: "misc"
```

Comment:
misc?

Reply:
This is not related to the GitHub page that gets created to I'm adding them to another directory